packages/sources/intrinio-test/src/endpoint/util.ts

```diff
@@ -0,0 +1,325 @@
+/**
+ * This code is directly copied from Intrinio's library.  We have a ticket to refactor this file.
```

Comment:
Do you know the ticket ID?

Reply:
no, it was a copy paste of the adapter v2, I don't have the ticket number mentioned in the comment